### ISSUE TITLE
Next and Previous buttons cannot be operated via keyboard.

### ISSUE THREAD

**Issue by sarbbottam** [gpbl/react-day-picker#430]
The [next and previous buttons](https://github.com/gpbl/react-day-picker/blob/b8184a65b39f74cbbed5bfaaba693812a4bdb972/src/Navbar.js#L38-L66) are implemented using `<span>`, thus they are not operable via `enter` or `space` press. Please consider it replacing with `<button>`.

Pros
- `tabindex=0` is no more needed, `<button>` is a focusable element
- `role=button` is no more needed, `<button>` exposes the role by default
-  `previousClickHandler` and `nextClickHandler` will get invoked when the buttons are activated using keyboard.

Cons
- might need to update the styles

**Comment by gpbl** [comment]
An old implementation still biting the component 👎 I think the best workaround would be to add `onKeyDown` event on it, so we can skip the major release. Thanks for the report!

**Comment by gpbl** [comment]
Fixed in v6.04.
